authors/settings.py

```diff
@@ -145,3 +145,12 @@
         'authors.apps.authentication.backends.JWTAuthentication',
     ),
 }
+
+# Sendgrid settings
+SEND_GRID_API_KEY = os.environ['SENDGRID_KEY']
```

Comment:
:thinking:  You are sending emails using the build in send_mail method. Idk if this is necessary